tidb-binlog/pump_client/client.go

```diff
@@ -321,42 +323,42 @@ func (c *PumpsClient) WriteBinlog(binlog *pb.Binlog) error {
 	log.Info("[pumps client] write binlog to avaliable pumps all failed, will try unavaliable pumps")
 	pump, err1 := c.backoffWriteBinlog(req, binlog.Tp, binlog.StartTs)
 	if err1 == nil {
+		choosePump = pump
```

Comment:
look like wrongly set before

Reply:
why change this? err1 is nil means write binlog to this pump success

Reply:
ignore it